### DISCUSSION TITLE
New Install Brewfile

### DISCUSSION THREAD

**Discussion by jrock2004** [nicknisi/dotfiles#174] [General]
I just reinstalled Mac on intel and brew seems to have changed something because nothing works with brew. Anyone else seeing that or is it just me? Issue was the install command has changed to install brew. Seems like casks command is no longer.

**Comment by nicknisi** [comment]
Ooh, interesting! I'm running both a ARM and Intel Mac daily but I haven't tried reinstalling on either recently. What command is failing? `brew bundle`?

**Reply by jrock2004** [reply]
Had to change how I install it to this:

```
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

The old way I get python errors. I still have not figured out but cask was not working much at all. You no longer need to tap cask but I am not sure. I just manually installed my tasks.
